solidity/test/StandardPoolConverter.js

```diff
@@ -1188,4 +1194,374 @@ describe('StandardPoolConverter', () => {
             });
         }
     });
+
+    describe('verifies that the network fee is transferred correctly via', () => {
+        const ONE_TOKEN = new BN(10).pow(new BN(18));
+        const TOTAL_SUPPLY = ONE_TOKEN.muln(1000000);
+        const CONVERSION_AMOUNT = ONE_TOKEN.muln(100);
+
+        for (const initialBalance1 of [100000, 200000, 400000, 800000]) {
+            for (const initialBalance2 of [100000, 300000, 500000, 700000]) {
+                for (const conversionFeePercent of [0, 5, 10, 25, 75]) {
+                    for (const networkFeePercent of [0, 5, 10, 25, 75, 100]) {
+                        it(description('processNetworkFees when', initialBalance1, initialBalance2, conversionFeePercent, networkFeePercent), async () => {
+                            const { poolToken, reserveToken1, reserveToken2, converter } = await createPool(networkFeePercent, conversionFeePercent);
+                            await addLiquidity(reserveToken1, reserveToken2, converter, [initialBalance1, initialBalance2].map(n => ONE_TOKEN.muln(n)));
+
+                            const conversion = await convert(reserveToken1, poolToken, reserveToken2, bancorNetwork, converter, CONVERSION_AMOUNT);
+                            const expectedFeeBase = conversion.fee.divn(2).muln(networkFeePercent).divn(100 + networkFeePercent);
+                            const reserveBalance1 = ONE_TOKEN.muln(initialBalance1).add(CONVERSION_AMOUNT);
+                            const reserveBalance2 = ONE_TOKEN.muln(initialBalance2).sub(conversion.amount);
+
+                            await converter.processNetworkFees();
+
+                            const balance1After = await reserveToken1.balanceOf(networkFeeWallet);
```

Comment:
Seems like `balance1After` and `balance2After` aren't used, here and below.

Reply:
Fixed.